geonode_mapstore_client/client/themes/geonode/scss/geonode.scss

```diff
@@ -0,0 +1,310 @@
+/*
```

Comment:
@allyoucanmap maybe I did something wrong when updating my branch but is this file needed

Reply:
@MuhweziDeo I think you can remove this file